napari/benchmarks/benchmark_slicing.py

```diff
@@ -0,0 +1,169 @@
+# See "Writing benchmarks" in the asv docs for more information.
+# https://asv.readthedocs.io/en/latest/writing_benchmarks.html
+# or the napari documentation on benchmarking
+# https://github.com/napari/napari/blob/main/docs/BENCHMARKS.md
+
+import time
+
+import numpy as np
+import zarr
+from qtpy.QtWidgets import QApplication
+
+import napari
+from napari.layers import Image
+
+SAMPLE_PARAMS = {
+    'skin_data': {
+        # napari-bio-sample-data
+        'shape': (1280, 960, 3),
+        'chunk_shape': (512, 512, 3),
+        'dtype': 'uint8',
+    },
+    'jrc_hela-2 (scale 3)': {
+        # s3://janelia-cosem-datasets/jrc_hela-2/jrc_hela-2.n5
+        'shape': (796, 200, 1500),
+        'dtype': 'uint16',
+        'chunk_shape': (64, 64, 64),
+    },
+}
+
+
+def get_image_params():
+    # chunksizes = [(64,64,64), (256,256,256), (512,512,512)]
+    latencies = [0.05 * i for i in range(0, 3)]
```

Comment:
I assume these latencies are in seconds, but clarify with a comment or more descriptive variable name (e.g. `latencies_secs`).

Also, is this latency per-chunk or per-call to `__getitem__`?

Reply:
Per `__getitem__`. As mentioned last week, I think it may be worth doing this via `dask` for a real world example (w/ chunk caching).